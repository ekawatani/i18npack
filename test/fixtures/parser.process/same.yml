foo: foo_text
bar: "{{ foo }} and {{ foo }}"