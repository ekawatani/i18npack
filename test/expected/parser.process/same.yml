foo: foo_text
bar: foo_text and foo_text